ui/src/kapacitor/components/KapacitorForm.tsx

```diff
@@ -95,25 +95,26 @@ const KapacitorForm: SFC<Props> = ({
                       inputType="password"
                     />
                   </div>
-                  {url.startsWith('https') && (
-                    <div className="form-group col-xs-12">
-                      <div className="form-control-static">
-                        <input
-                          type="checkbox"
-                          id="insecureSkipVerifyCheckbox"
-                          name="insecureSkipVerify"
-                          checked={insecureSkipVerify}
-                          onChange={onCheckboxChange}
-                        />
-                        <label htmlFor="insecureSkipVerifyCheckbox">
-                          Unsafe SSL
+                  {url &&
```

Comment:
maybe some kind of typechecking, prior guard, or default value could be used to ensure url exists? why would we be able to get here without url?

Reply:
It was getting a null value right after I saved a new kapacitor configuration.

Reply:
I think this component needs to be translated to a class based component, there are enough places that need default values to make it a cleaner experience. However this change was not in service of the issue so didn't want to bog down this PR too much.

Reply:
hm. i don't think a url should ever be null after saving a new kapacitor configuration. i suspect introducing this line will cover up part of what, i would agree, should be a larger refactor. could you make a chore issue for this and cite this line and the `null` upon saving for future consideration?

Reply:
sounds good